ScratchDjango/User/managers.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth.models import UserManager as DjangoUserManager
+from django.contrib.auth.password_validation import validate_password
+
+
+class UserManager(DjangoUserManager):
+    def _create_user(self, email: str, password: str | None, **extra_fields):
+        """
+        Create and save a user with the given email and password.
+        """
+        if not email:
+            raise ValueError("The given email must be set")
+        validate_password(password)
```

Comment:
What if password is not validated?

Reply:
It will throw the exception which will be catched  by the register view and will show that as message to the user